localization/ndt_scan_matcher/src/ndt_scan_matcher_core.cpp

```diff
@@ -281,6 +281,13 @@ void NDTScanMatcher::timer_diagnostic()
       diag_status_msg.level = diagnostic_msgs::msg::DiagnosticStatus::ERROR;
       diag_status_msg.message += "skipping_publish_num exceed limit. ";
     }
+    if (
+      state_ptr_->count("nearest_voxel_transformation_likelihood") &&
+      std::stod((*state_ptr_)["nearest_voxel_transformation_likelihood"]) <
+        converged_param_nearest_voxel_transformation_likelihood_) {
+      diag_status_msg.level = diagnostic_msgs::msg::DiagnosticStatus::WARN;
+      diag_status_msg.message += "NDT score is unreliably low. ";
+    }
```

Comment:
Just noticed that this is unnecessary.

Since `skipping_num` is dependent on `is_converged` according to [L433-L440](https://github.com/autowarefoundation/autoware.universe/blob/0ab6a3be3835f97b87c524896aca9d9ab75cb92d/localization/ndt_scan_matcher/src/ndt_scan_matcher_core.cpp#L433-L440), the `skipping_publish_num` criteria in diagnostics already captures whether the latest NDT results is converged or not.

So IMHO no need to merge this PR.

Reply:
Understood, thanks!